ts/packages/node/src/stores/ORM.ts

```diff
@@ -1,59 +1,72 @@
 import 'reflect-metadata';
 import { ConfigStore, type ConfigStoreQuery, type Config as IConfig } from '@configu/ts';
-import { Entity, Column, DataSource, type DataSourceOptions, Index, PrimaryGeneratedColumn } from 'typeorm';
+import { DataSource, type DataSourceOptions, Entity, Index, PrimaryGeneratedColumn, Column } from 'typeorm';
 import _ from 'lodash';
 
-@Entity()
-@Index(['set', 'key'], { unique: true })
-class Config {
-  @PrimaryGeneratedColumn('uuid')
-  id: string;
+const createEntity = (tableName: string) => {
+  @Entity({ name: tableName })
+  @Index(['set', 'key'], { unique: true })
+  class Config {
+    @PrimaryGeneratedColumn('uuid')
+    id: string;
 
-  @Index('set')
-  @Column('text')
-  set: string;
+    @Index(`${tableName}_set`)
+    @Column('text')
+    set: string;
 
-  @Column('text')
-  key: string;
+    @Column('text')
+    key: string;
 
-  @Column('text')
-  value: string;
-}
+    @Column('text')
+    value: string;
+  }
+
+  return Config;
+};
+
+type ORMConfigStoreOptions = DataSourceOptions & {
+  tableName?: string;
+};
 
 export abstract class ORMConfigStore extends ConfigStore {
   readonly dataSource: DataSource;
+  private readonly table: ReturnType<typeof createEntity>;
```

Comment:
I think the disconnect between the class property name and the parameter `tableName` is a source of confusion. I suggest that we align the naming either way.